File: Python/URI/1066.py
positivo = 0
negativo = 0
par = 0
impar = 0
for i in range(5):
    numero = int(input())


    if numero > 0: 
        positivo+=1
    
    if numero < 0:
        negativo+=1
    
    if numero % 2 == 0:
        par+=1
    
    if numero % 2 != 0:
        impar+=1
    




print(str(par)+" valor(es) par(es)")
print(str(impar)+" valor(es) impar(es)")
print(str(positivo)+" valor(es) positivo(s)")
print(str(negativo)+" valor(es) negativo(s)")

File: JS/URI/1066.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');


var positivo = 0;
var negativo = 0;
var par = 0;
var impar = 0;
for (var i = 0; i < 5; i++) {
    var numero = parseFloat(lines.shift());


    if (numero > 0) {
        positivo++;
    }
    if (numero < 0) {
        negativo++;
    }
    if (numero % 2 == 0) {
        par++;
    }
    if (numero % 2 != 0) {
        impar++;
    }


}

console.log(par + " valor(es) par(es)");
console.log(impar + " valor(es) impar(es)");
console.log(positivo + " valor(es) positivo(s)");
console.log(negativo + " valor(es) negativo(s)");
